### DISCUSSION TITLE
blog/2021-03-19-virtualmachine-to-docker/

### DISCUSSION THREAD

**Discussion by giscus** [crazychenz/vinnie.work#28] [vinnie.work Comments]
# blog/2021-03-19-virtualmachine-to-docker/

I was recently tasked at work with making some routine enhancements to an internal project that was built with toolchains that had been organized into Virtual Machines that I had to fire up. With my recent adoption of Docker as a mainline tool in my everyday work flow, I asked myself: *Why the heck am I using a hypervisor for a straight forward toolchain VM?*

https://www.vinnie.work/blog/2021-03-19-virtualmachine-to-docker/

**Comment by Jaffa609** [comment]
Not too many steps through just yet however I have noticed what I believe to be a typo. Under 'Converting OVA to Raw Data' there is a line such as: 'qemu-img -f vmdk MyDisk.vmdk -O raw MyDisk.raw'. Is the word 'convert' not missing from here.

**Reply by crazychenz** [reply]
You are correct. Thank you for the correction!
